### PR TITLE
165404349 users can update their articles

### PR DESCRIPTION
**Pull request by Jesse-efe**
#### Description
users can update articles they authored
#### Type of change
Server
#### How Has This Been Tested?
tests were written and they all pass
#### Checklist:
N/A
#### Pivotal tracker
#165404349
#### Flow chart
update article

### CODE REVIEW

**Review by Fiyiin — COMMENTED** [review]
LGTM

File: test/articleTest.js [hunk 1]
```diff
@@ -251,6 +251,28 @@ describe('article like', () => {
...
+  it('should update an article', async () => {
+    const result = await server.patch(`/api/v1/articles/${articleId}`)
+      .set('authorization', testToken)
+      .send({
+        body: 'it is not good',
+        plainText: 'jsjfosdf'
+      }).expect(201);
+    expect(result.status).to.equal(201);
+  });
+});
```
Comment (Joyce-O) [inline test/articleTest.js:300]: Hey, good job on capturing the edge cases in the test, but kindly look into the *it* blocks as one is a duplicate of the other.

**Review by Joyce-O — CHANGES_REQUESTED** [review]

File: test/articleTest.js (hunk 1 above)
Comment (Jesse-efe) [reply test/articleTest.js:300]: would do so ..thanks

**Review by Joyce-O — APPROVED** [review]

**Comment by Joyce-O** [comment]
LGTM
